custom_components/ocpp/switch.py

```diff
@@ -56,7 +56,15 @@ class OcppSwitchDescription(SwitchEntityDescription):
         on_action=HAChargerServices.service_availability.name,
         off_action=HAChargerServices.service_availability.name,
         metric_state=HAChargerStatuses.status_connector.value,
-        metric_condition=[ChargePointStatus.available.value],
+        metric_condition=[
```

Comment:
> It also enables the switch to be turned on in these other states.

Whereby nothing happens, I presume, because the charger was already enabled and will still have the vehicle connected?

I'll just revert this change. It's mostly there because this whole thing started as an internal demo before being promoted to upstream contribution. I don't actually feel any need to change this behaviour right now.